sdk/servicebus/service-bus/test/stress/scenarioLongRunning.ts

```diff
@@ -0,0 +1,105 @@
+import {hostname} from "os";
+import { captureConsoleOutputToAppInsights, defaultClient, SBStressTestsBase } from "./stressTestsBase";
+import { AbortController, AbortSignalLike } from "@azure/abort-controller";
+import { ServiceBusClient, ServiceBusSender } from "@azure/service-bus";
+import { v4 as uuidv4 } from "uuid";
+
+captureConsoleOutputToAppInsights();
+
+const stressTest = new SBStressTestsBase({
+    testName: "longRunning",
+    snapshotFocus: [
+        "send-info",
+        "receive-info"
+    ]
+});
+
+async function looper(fn: () => Promise<void>, delay: number, abortSignal: AbortSignalLike) {
+    const timeout = () => new Promise((resolve) => setTimeout(() => resolve(true), delay));
+
+    while (!abortSignal.aborted && await timeout()) {
+        await fn();
+    }
+}
+
+async function sendMessagesForever(clientForSender: ServiceBusClient, abortSignal: AbortSignalLike) {
+    console.log(`Started message sending`);
+
+    let sender: ServiceBusSender|undefined;
```

Comment:
```suggestion
    let sender: ServiceBusSender | undefined;
```
Why didn't the formatter complain?

Reply:
I'll have to check in on that. I'm not sure why - perhaps it's the sub-sub-project nature of this particular folder.